letsencrypt-nginx/letsencrypt_nginx/tests/configurator_test.py

```diff
@@ -14,7 +14,6 @@
 
 from letsencrypt_nginx.tests import util
 
-
```

Comment:
FYI: Two blank lines before classes or class level functions is standard pep8.
https://www.python.org/dev/peps/pep-0008/#blank-lines